.github/workflows/pixi-ci.yml

```
@@ -16,6 +27,15 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
+    # On periodic jobs and when delete_pixi_lock option is true, delete the pixi.lock to check that the project compiles with latest version of dependencies
+    - name: Delete pixi.lock on scheduled jobs or if delete_pixi_lock is true
+      if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.delete_pixi_lock == 'true')
+      shell: bash
+      run: |
+        rm -rf pixi.lock
+        pixi list
+        cat pixi.lock
```

this won't work since pixi is not installed at that time. you can only remove pixi.lock and setup-pixi will install and run `pixi list` instead.

```suggestion
        rm pixi.lock
```